setpkgs.sh

```
@@ -13,8 +13,55 @@ else
     SAB_CMD="cpulimit -l $CPU_LIMIT -- $SABNZBDPLUS"
 fi
 
-tracker_list=$(curl -Ns https://ngosang.github.io/trackerslist/trackers_all_http.txt | awk '$0' | tr '\n\n' ',')
-$ARIA2_CMD --conf-path=configs/ac/ac.conf --daemon=true --rpc-listen-all=true --bt-tracker="[$tracker_list]"
+tracker_list=$(curl -Ns https://cdn.jsdelivr.net/gh/ngosang/trackerslist@master/trackers_all.txt | awk '$0' | tr '\n\n' ',')
```

Shellcheck (0.11.0)

[info] tr replaces sets of chars, not words (mentioned due to duplicates).

(SC2020)